Lexis+.js

```diff
@@ -0,0 +1,267 @@
+{
+	"translatorID": "419638d9-9049-44ad-ba08-fa54ed24b5e6",
+	"label": "Lexis+",
+	"creator": "bfahrenfort",
+	"target": "^https://plus.lexis.*/",
```

Comment:
```suggestion
	"target": "^https?://plus\\.lexis\\..*/",
```

These are not all lexis.com, I take it?